src/schema.ts

```diff
@@ -25,7 +25,7 @@ export const typeDefs = `#graphql
   }
 
   type Query {
-    hello: String
+    user(id: ID): User
```

Comment:
```suggestion
    user(id: ID!): User
```

O ponto de exclamação é pra indicar que o input é obrigatório, que parece ser o seu caso pelo que eu entendi da implementação aqui.